### PR TITLE
Load env vars from file

### LINKED ISSUE #76: Direct support for an env file when launching the app
Startup when env file right now works only if the env file is used in docker to mock env vars. Modify the main file instead to accept and use and env file also when started from conda env.

### PR DESCRIPTION
**Pull request by northwestwitch**
### This PR adds | fixes:
- Fix #76 -> Include env vars in a .env file to be loaded when app is launched. This is to allow database connections on local machine without having to set hardcoded env vars on the system itself.

### How to test:
- Start a demo instance without editing the .env file
- Given a local MySQL database, edit the .env file to connect to it.
- Start a demo instance using docker-compose 
- Start the app using a container connected to a mariadb container

### Expected outcome:
- [x] All 4 instances should start and be connected to the intended databases

### Review:
- [x] Code approved by HS
- [x] Tests executed by CR

This [version](https://semver.org/) is a:
- [ ] **MAJOR** - when you make incompatible API changes
- [x] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

### CODE REVIEW

**Comment by northwestwitch** [comment]
Demo instance from conda env: 🆗 

image


Connecting to a local database (named `chanjo2_test`) after modifying the .env file: 🆗 

image

Starting demo app from docker-compose (command: `docker-compose up`): 🆗 

image

Starting a chanjo2 from docker-compose with connection to a dockerized database (command: `docker-compose -f docker-compose-mysql.yml --env-file template.env up`): 🆗 

image

File: src/chanjo2/__init__.py [hunk 1]
```diff
@@ -1 +1,4 @@
+from dotenv import load_dotenv
+
 __version__ = "0.1.0"
+load_dotenv()
```
Comment (northwestwitch) [inline src/chanjo2/__init__.py:4]: This is the place where the ENV vars are loaded from the .env file

**Comment by northwestwitch** [comment]
@henrikstranneheim if you have some time to review this is an easy one!

**Review by henrikstranneheim — APPROVED** [review]
Nice 👍

**Pull request merged**